pytapo/__init__.py

```diff
@@ -1086,6 +1086,80 @@ def getWhitelampStatus(self):
             "getWhitelampStatus", {"image": {"get_wtl_status": ["null"]}}
         )
 
+    ### TODO: UNTESTED
+    def getFloodlightStatus(self):
+        return self.executeFunction(
+            "getFloodlightStatus", {"floodlight": {"get_floodlight_status": ""}}
+        )
```

Comment:
If it works, we need to probably extract some specific property, just like in other functions